config/analytics_blocklist.yml

```diff
@@ -93,11 +102,10 @@
   :emergency_login_keys:
   - id
   - not_valid_after
-  - publisher_id
```

Comment:
related to this PR? Unrelated? Heads up in case we're bringing an unexpected change to `main` branch.